data/json/items/comestibles/junkfood.json

```diff
@@ -1288,5 +1330,152 @@
     "flags": [ "EATEN_HOT" ],
     "vitamins": [ [ "vitA", 3 ], [ "iron", 10 ] ],
     "fun": 4
+  },
+  {
+    "type": "COMESTIBLE",
+    "id": "gelatin_dessert_powder_base",
+    "name": "gelatin dessert powder",
+    "weight": "175 g",
+    "color": "white",
+    "spoils_in": "1461 days",
+    "container": "box_small",
+    "comestible_type": "FOOD",
+    "symbol": "%",
+    "quench": -8,
+    "calories": 20,
+    "description": "A small box of gelatin dessert powder. Just add water and set. Comes in a variety of flavors.",
+    "price": 5,
+    "price_postapoc": 10,
+    "material": [ "flesh", "junk" ],
+    "volume": "250ml",
+    "flags": [ "EATEN_COLD" ],
+    "fun": -2
+  },
+  {
+    "type": "COMESTIBLE",
+    "id": "gelatin_dessert_powder_cherry",
+    "name": "cherry gelatin dessert powder",
+    "color": "red",
+    "description": "A small box of cherry gelatin dessert powder. Just add water and set.",
+    "copy-from": "gelatin_dessert_powder_base"
+  },
+  {
+    "type": "COMESTIBLE",
+    "id": "gelatin_dessert_powder_lime",
+    "name": "lime gelatin dessert powder",
+    "color": "light_green",
+    "description": "A small box of lime gelatin dessert powder. Just add water and set.",
+    "copy-from": "gelatin_dessert_powder_base"
+  },
+  {
+    "type": "COMESTIBLE",
+    "id": "gelatin_dessert_powder_grape",
+    "name": "grape gelatin dessert powder",
+    "color": "magenta",
+    "description": "A small box of grape gelatin dessert powder. Just add water and set.",
+    "copy-from": "gelatin_dessert_powder_base"
+  },
+  {
+    "type": "COMESTIBLE",
+    "id": "gelatin_dessert_powder_raspberry",
+    "name": "blue raspberry gelatin dessert powder",
+    "color": "cyan",
+    "description": "A small box of blue raspberry gelatin dessert powder. Just add water and set.",
+    "copy-from": "gelatin_dessert_powder_base"
+  },
+  {
+    "type": "COMESTIBLE",
+    "id": "gelatin_dessert_base",
+    "name": "unflavored gelatin dessert",
+    "weight": "232 g",
+    "color": "white",
+    "spoils_in": "7 days 12 hours",
+    "container": "cup_plastic",
+    "comestible_type": "FOOD",
+    "symbol": "%",
+    "calories": 100,
+    "description": "An unflavored jiggly, sugary treat made from gelatin and sugar. This would taste better with some flavor.",
+    "price": 210,
+    "price_postapoc": 250,
+    "material": [ "flesh", "junk" ],
+    "volume": "250 ml",
+    "phase": "solid",
+    "fun": 5
+  },
+  {
+    "type": "COMESTIBLE",
+    "id": "gelatin_dessert_cherry",
+    "name": "cherry gelatin dessert",
+    "color": "red",
+    "description": "A cherry flavored jiggly, sugary treat made from gelatin and sugar. A kid favorite pre-Cataclysm.",
+    "fun": 20,
+    "copy-from": "gelatin_dessert_base"
+  },
```

Comment:
This and the other _flavored_ gelatins compressed into snippets for one, as well.